Assets/HoloToolkit-Examples/Input/Scripts/ButtonObjectScaler.cs

```diff
@@ -48,22 +50,28 @@ private void OnButtonPressed(TestButton source)
                 switch (ButtonAction)
                 {
                     case buttonAction.Reset:
-                        Debug.Log(InitialScale);
-                        ObjectToScale.transform.localScale = InitialScale;
-                        break;
+                        {
+                            Debug.Log(InitialScale);
+                            ObjectToScale.transform.localScale = InitialScale;
+                            break;
+                        }
                     case buttonAction.Grow:
-                        ObjectToScale.transform.localScale = new Vector3((ObjectToScale.transform.localScale.x + ScaleIncrement), (ObjectToScale.transform.localScale.y + ScaleIncrement), (ObjectToScale.transform.localScale.z + ScaleIncrement));
-                        break;
-                    case buttonAction.Shrink:
```

Comment:
This seems to be more than just documentation changes.

Reply:
I had done no changes regarding the coding as I only added some comments in the coding part. Besides, some of the coding are out of what I had learned so I won't touch those coding, sorry for making any inconvenience.

Reply:
I remembered that I had added {} to those loops, is it ok?